src/FixedDataTable.js

```diff
@@ -1037,6 +1060,8 @@ class FixedDataTable extends React.Component {
     this._divRef = div;
     if (this.props.stopReactWheelPropagation) {
       this._wheelHandler.setRoot(div);
+    } else {
+      this._wheelHandler.setRoot(null);
```

Comment:
Picked this up along the way.
Not exactly sure how we'll hit this code, but logically this just resets the root node if the prop is also reset.